recipe/meta.yaml

```diff
@@ -31,17 +31,25 @@ test:
   requires:
     - pip
     - pytest
+    - flake8
+    - pytest
+    - codecov
+    - coverage
+    - pytest-cov
+    - pytest-env
+    - pytest-mock
```

Comment:
yeah, we probably don't need most of it here since we aren't really reporting our coverage. could you please only add the one that was newly added?

Reply:
is this only `pytest-mock`?

Reply:
yes, https://github.com/diffpy/diffpy.fourigui/commit/c657f237613ef6d542ed8dc7dc55022c08f80b07